network/p2p/gossip/bloom.go

```diff
@@ -5,27 +5,22 @@ package gossip
 
 import (
 	"crypto/rand"
-	"encoding/binary"
-	"hash"
-
-	bloomfilter "github.com/holiman/bloomfilter/v2"
 
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/bloom"
 )
 
-var _ hash.Hash64 = (*hasher)(nil)
-
 // NewBloomFilter returns a new instance of a bloom filter with at most
 // [maxExpectedElements] elements anticipated at any moment, and a false
 // positive probability of [falsePositiveProbability].
 func NewBloomFilter(
```

Comment:
Should we note that this isn't safe for concurrent use? (could Reset while Marshaling)